x-pack/test/security_solution_cypress/config.ts

```diff
@@ -49,6 +49,7 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
         `--xpack.securitySolution.enableExperimental=${JSON.stringify([
           'alertDetailsPageEnabled',
           'chartEmbeddablesEnabled',
+          'riskScoringRoutesEnabled',
```

Comment:
As we are enabling the FF on the test, we can remove it from here.

Reply:
By some reason, removing this make test failed, and looks like, enabling FF in test doesn't work(i removed from test, and keep it here)